<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SDN-93" ignored="0" name="MemberCase-Tests" tests="1" failures="1" timestamp="18 Nov 2019 11:41:12 GMT" time="33.508" errors="0">
  <testcase name="verifyAddEnquiry" time="21.853" classname="AddEnquiry">
    <failure type="java.lang.AssertionError" message="expected [Enquiry Added Successfully] but found []">
        <![CDATA[java.lang.AssertionError: expected [Enquiry Added Successfully] but found []
	at AddEnquiry.verifyAddEnquiry(AddEnquiry.java:47)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyAddEnquiry -->
</testsuite> <!-- S_MemberCase-Tests -->
